code/pagetypes/Forum.php

```diff
@@ -109,7 +109,7 @@ function canModerate($member = null) {
 		if(!$member) return false;
 		
 		// Admins
-		if ($this->canEdit($member)) return true; 
+		if ($member->inGroup('administrators')) return true; 
```

Comment:
Also I'd add

``php
$extended = $this->extendedCan('canModerate', $member);
if($extended !== null) return $extended;
```